scripts/protobuf_codegen.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
```

Comment:
Why do we set this flag?

Reply:
No functional reason, @hexfusion asked to keep it in since it echos every operation performed by the script